docs/source/changelog/0.17.0-changelog.rst

```diff
@@ -0,0 +1,265 @@
+
+
+* :pr:`2915`: Added documentation to :mod:`.three_dimensions`
+
+
+* :pr:`2919`: Corrected parameters and enhanced the description of :meth:`.ImageMobject.interpolate_color`
+
+
+* :pr:`2932`: Fixed whitespace formatting issue
+
+
+* :pr:`2933`: Improved answer to the "no scenes in this module" error
+
+
+* :pr:`2936`: Added installation instructions for Windows via ``winget``
+
+
+* :pr:`2962`: Disabled "Edit on GitHub" button in documentation
+
+
+* :pr:`2978`: Added documentation and example for :class:`.CyclicReplace`
+
+
+* :pr:`3001`: Added FAQ entry regarding failed ``manimpango`` build
+
+
+* :pr:`3004`: Fixed docbuild warnings
+
+
+* :pr:`3018`: Follow-up to :pr:`2988` -- fixes and improvements to some docstrings
+
+
+* :pr:`3022`: Corrected type hint in :meth:`Axes.coords_to_point`
+
+
+Changes concerning the testing system
+-------------------------------------
+
+* :pr:`2893`: Improved performance of ``test_threed.py``
+
+
+* :pr:`2981`: Implemented fallback save behavior for ``pytest --show_diff``
+
+
+* :pr:`2982`: Rewrote unstable tests for :mod:`.text_mobject` to be non-graphical
+
+
+* :pr:`2991`: Migrate ``os.path`` to ``pathlib.Path`` in tests
+
+
+Changes to our development infrastructure
+-----------------------------------------
+
+* :pr:`2925`: CI (test-arm): update python version to `3.10.6`
+
+
+* :pr:`2963`: CI (test-arm): Always select the correct python version
+
+
+* :pr:`3029`: CI: update actions version and add dependabot config
+
+
+Code quality improvements and similar refactors
+-----------------------------------------------
+
+* :pr:`2872`: Add ``extract_frames.py`` utility script to help visualize test control data
+
+
+* :pr:`2877`: Fixed binder launch problem by adding missing optional ``notebook`` dependency
+
+
+* :pr:`2887`: Removed empty i18n files that caused filename clashes
+
+
+* :pr:`2931`: Updated ``mapbox-earcut``
+
+
+* :pr:`2938`: Suggested fixes by iCR, OpenRefactory, Inc.
+
+
+* :pr:`2954`: Fixed click version string in ``pyproject.toml``
+
+
+* :pr:`2958`: Fix missing stub packages for mypy
+
+
+* :pr:`2975`: Fixed broken links in README
+
+
+* :pr:`2980`: Migrate more ``os.path`` to ``pathlib.Path``
+
+
+* :pr:`2983`: Fixed Windows CI Pipeline
+
+
+* :pr:`2988`: Converted all types of parameters in docstrings to proper type annotations
+
+
+* :pr:`2994`: Fixed segmentation faults from doctests under Python 3.10
+
+
+* :pr:`2995`: Added encoding to ``open`` in :mod:`.utils.text_file_writing`
+
+
+* :pr:`3032`: Bump jupyter-core from 4.11.1 to 4.11.2
+
+
+* :pr:`3033`: Bump pillow from 9.2.0 to 9.3.0
+
+
+New releases
+------------
+
+* :pr:`3023`: Prepared new release: v0.17.0
+
+
+Unclassified changes
+--------------------
+
+* :pr:`3035`: Include latex install instructions on ubuntu
+
+
+* :pr:`3044`: [docs] Add Debian dependencies required for pycairo and manimpango
+
+
+* :pr:`3045`: Update python-opengl -> python3-opengl for Ubuntu CI task
```

Comment:
I'd classify this as "Changes to our development infrastructure"

Reply:
Done, labeled https://github.com/ManimCommunity/manim/pull/3045 as infrastructure (and #3035, #3044 as documentation). Will be reflected in the next changelog generation pass.